Common/Core/RecoDecay.h

```diff
@@ -908,25 +908,25 @@ class RecoDecay
           }
         }
         if (!isPDGFound) {
-          //Printf("MC Gen: Rejected: bad daughter PDG: %d", PDGCandidateDaughterI);
+          // Printf("MC Gen: Rejected: bad daughter PDG: %d", PDGCandidateDaughterI);
           return false;
         }
       }
       if (listIndexDaughters) {
         *listIndexDaughters = arrAllDaughtersIndex;
       }
     }
-    //Printf("MC Gen: Accepted: m: %d", candidate.globalIndex());
+    // Printf("MC Gen: Accepted: m: %d", candidate.globalIndex());
     if (sign) {
       *sign = sgn;
     }
     return true;
   }
 
-  /// Finds the origin (from charm hadronisation or beauty-hadron decay) of charm hadrons.
+  /// Finds the origin (from charm hadronisation or beauty-hadron decay) of charm hadrons. It can be used also to verify whether a hadron derives from charm or beauty decay.
```

Comment:
```suggestion
  /// Finds the origin (from charm hadronisation or beauty-hadron decay) of charm hadrons. It can be used also to verify whether a particle derives from a charm or beauty decay.
```
Would this be fine @arossi81 ?

Reply:
Hi @vkucera ,
yes, that's ok!
Cheers,
Andrea